packages/metals-vscode/src/testExplorer/testRunHandler.ts

```diff
@@ -73,7 +73,11 @@ export async function runHandler(
     for (const test of tests) {
       if (!excludes.has(test)) {
         const kind = test._metalsKind;
-        if (kind === "project" || kind === "package") {
+        if (
+          kind === "project" ||
+          kind === "package" ||
+          kind === "workspaceFolder"
+        ) {
```

Comment:
I should have write some documentation about this :( Consider adding my loud thoughts as comments/proper doc somewhere.
there is no need to change more bits here because `createRunQueue` call itself recursively, so almost nothing changed in terms of running - still kind `suite` is the operating atom here and in result analyzer.